### PR TITLE
changed colored back pressed notification

### LINKED ISSUE #2290: changed colored back pressed notification
**Actual Behaviour**
Unnecessary snackbar shown while exiting the app

**Expected Behaviour**
Adding colored toast notification when user press back to exit 
Toast will change color when primary color is changed from settings
UI will be more optimized

**Would you like to work on the issue?**
YES

### PR DESCRIPTION
**Pull request by smaliaquib**
Fixed #2290

Changes: 
Added colored toast notification when user press back to exit
Toast will change color when primary color is changed from settings
UI will be more optimized

Screenshots of the change: 
screenshot_2018-11-24-19-55-11-478_org fossasia phimpme

### CODE REVIEW

**Comment by abishekvashok** [comment]
Please rebase properly

**Comment by smaliaquib** [comment]
Done relevant changes and rebase properly please tell me if you need any changes in it

**Comment by smaliaquib** [comment]
made changes please approve it @abishekvashok  @codepoet2017390

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java [hunk 1]
```diff
@@ -3406,25 +3414,17 @@ public void onBackPressed() {
...
-                                params.topMargin,
-                                params.rightMargin,
-                                params.bottomMargin + navigationView.getHeight());
-                        sbView.setLayoutParams(params);
-                        snackbar.show();
 
+                        final View layouttoast;
+                        LayoutInflater inflater = getLayoutInflater();
+                        layouttoast = inflater.inflate(R.layout.toast_custom, (ViewGroup) findViewById(R.id.toastcustom));
+                        ((TextView) layouttoast.findViewById(R.id.texttoast)).setText("Press back again to exit");
```
Comment (abishekvashok) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: Please use the string resource

Comment (smaliaquib) [reply app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: Done

**Comment by abishekvashok** [comment]
Closing as obsolete

**Pull request closed without merging**